rs/ledger_suite/common/ledger_canister_core/BUILD.bazel

```diff
@@ -14,8 +14,8 @@ rust_library(
     deps = [
```

Comment:
[Adding this here, because GitHub only forces you to resolve inline comments, not top-level ones.]

You should make @dfinity/finint an owner of `rs/ledger_suite` in `.github/CODEOWNERS`. As is, it's @dfinity/ic-interface-owners that owns it (along with everything not explicitly owned under `rs`).

Reply:
Thanks, I edited the codeowner part.